fix: 2965 Correct Parsing Logic for Qualitative Instance Level SR

GitanjaliChhetri opened the pull request:
Fix for https://github.com/OHIF/Viewers/issues/2965

Explanation below:
Qualitative slice-level SR annotations are within the Measurement Group container, which is obtained through https://github.com/OHIF/Viewers/blob/7aba01f534b70406ce02e1708004141c7b773904/platform/core/src/DICOMSR/SCOORD3D/parseSCOORD3D.js#L31

This particular dataset did not have any ImageLibrary, hence [getReferenceImagesList()](https://github.com/OHIF/Viewers/blob/7aba01f534b70406ce02e1708004141c7b773904/platform/core/src/DICOMSR/SCOORD3D/parseSCOORD3D.js#L30) was throwing errors. 

I modified [platform/core/src/DICOMSR/SCOORD3D/utils/getReferencedImagesList.js](https://github.com/OHIF/Viewers/blob/7aba01f534b70406ce02e1708004141c7b773904/platform/core/src/DICOMSR/SCOORD3D/utils/getReferencedImagesList.js#L4) as it should not look for CodeValue from ImagingMeasurements.

sedghi reviewed (changes requested):
Please updat the description of the PR, I'm not in the loop of what is happening. Why we are removing referenced images?

fedorov commented:
I second @sedghi - @GitanjaliChhetri can you please summarize what this PR is doing and why?

GitanjaliChhetri commented:
> 

Updated the description to explain the fix,

fedorov commented:
@GitanjaliChhetri can you please include specific SR samples that you are using to guide the development in this PR and test the revised code?

sedghi commented:
I let @igoroctaviano review first

GitanjaliChhetri commented:
> @GitanjaliChhetri can you please include specific SR samples that you are using to guide the development in this PR and test the revised code?
> Included in my response.
> > Reference images are not required for slice level annotations, moreover it wasn't used anywhere in the code.
> 
> What do you mean by "reference images" here?
> I think 'referencedImages' at https://github.com/OHIF/Viewers/blob/7aba01f534b70406ce02e1708004141c7b773904/platform/core/src/DICOMSR/SCOORD3D/parseSCOORD3D.js#L30 is a slightly confusing object name because this object is returned from the Image Library Container within the TID 1600 template.
This is not the IMAGE content item within the Measurement group container.
> > I reverted his changes from [platform/core/src/DICOMSR/SCOORD3D/utils/getReferencedImagesList.js](https://github.com/OHIF/Viewers/blob/7aba01f534b70406ce02e1708004141c7b773904/platform/core/src/DICOMSR/SCOORD3D/utils/getReferencedImagesList.js#L4) as it should not look for CodeValue from ImagingMeasurements.
> 
> I am not sure I understand what this means either ("as it should not look for CodeValue from ImagingMeasurements").
>>I meant that while searching the SR content tree for the Image Library Container, we should not be looking into the of Imaging Measurements container. Hence I reverted that portion of the fix.

igoroctaviano reviewed (approved):
LGTM
